### ISSUE TITLE
Support --ignore flag for unused keys

### ISSUE THREAD

**Issue by ceolinwill** [lingualdev/i18n-check#90]
The `--ignore` / `-i` flag is very helpful for missing keys, but it doesn’t currently work with the unused keys check.

**Example:**

```bash
pnpm i18n-check -s en -l messages --only unused -u app -f next-intl -i "Menu.*"
```

Even though `Menu.*` is passed to `--ignore`, those keys are still reported as unused.

**Motivation:**
There are scenarios where some keys may appear unused (e.g. dynamic references) but we don’t want them flagged. Allowing `--ignore` to apply to unused keys would provide a clean way to handle these cases.

**Related issues:**
This could also serve as a workaround for #89.

**Comment by busypeoples** [comment]
We will open a PR for this one, it makes to support it for the `unused` and `undefined` keys.
